src/FsCheck/Runner.fs

```diff
@@ -377,19 +380,27 @@ module Runner =
         let tryShrinkNb = ref 0
         let origArgs = ref []
         let lastStep = ref (Failed Res.rejectedV)
-        let seed = match config.Replay with None -> Random.create() | Some s -> s
+        let seed, size = match config.Replay with None -> Random.create(), None | Some s -> s.Rnd, s.Size
```

Comment:
It looks stupid to keep replay size in separate field when we already have `config.StartSize` but nevertheless we have to indicate that this is "replay run" somehow.